templates/vue/src/components/Lightbox.vue

```diff
@@ -1,18 +1,19 @@
 <template>
   <div
+    v-if="isTapestryLoaded"
```

Comment:
Can we change this to `tapestryLoaded` or `tapestryIsLoaded` instead since it will read nicer: "if tapestry is loaded"

Reply:
Done, thanks.